code/modules/client/preferences.dm

```diff
@@ -1205,10 +1205,16 @@ var/const/MAX_SAVE_SLOTS = 10
 					commander_sidearm = new_co_sidearm
 
 				if("grade_path")
-					var/list/options = list("Command", "Technical")
+					var/list/options = list(SEA_RANKS[1])
+					if(get_job_playtime(owner, JOB_SEA) >= JOB_PLAYTIME_TIER_1)
+						options += list(SEA_RANKS[2]) + list(SEA_RANKS[3])
+					if(get_job_playtime(owner, JOB_SEA) >= JOB_PLAYTIME_TIER_3)
+						options += list(SEA_RANKS[4]) + list(SEA_RANKS[5])
+
 					var/new_path = tgui_input_list(user, "Choose your preferred promotion path.", "Promotion Paths", options)
```

Comment:
```suggestion
					var/new_path = tgui_input_list(user, "Choose your preferred paygrade.", "Paygrades", options)
```
Or
```suggestion
					var/new_path = tgui_input_list(user, "Choose your preferred rank.", "Ranks", options)
```
if you can make associated list of ranks-to-paygrades work.
Should probably be accompanied with a change to this, wherever the code for that is, too.
![image](https://user-images.githubusercontent.com/4447185/226179380-4e1a0c1b-2b79-46b8-a4c1-f046ebfe6fd7.png)